schemas/com.mparticle.snowplow/appstatetransition_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,40 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an app state transition event",
+	"self": {
+		"vendor": "com.mparticle.snowplow",
+		"name": "appstatetransition_event",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"transitionType": {
+			"type": "string"
+		},
+		"isFirstRun": {
+			"type": "boolean"
+		},
+		"isUpgrade": {
+			"type": "boolean"
+		},
+		"launchParams": {
+			"type": "string"
+		},
+		"launchReferral": {
+			"type": "string"
+		},
+		"successfullyClosed": {
+			"type": "boolean"
+		},
+		"jsonPayload": {
+			"type": "number"
```

Comment:
This sounds fishy - jsonPayload as a number?

Reply:
Yes good catch. This should be a string.